tools/cluster_mgr.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import math
+import redis
+import subprocess
+import time
+
+'''
+To install: pip install -r requirements.txt
+'''
+
+
+class Node:
+    def __init__(self, port, admin_port):
+        self.port = port
+        self.admin_port = admin_port
+
+    id = ''
+
```

Comment:
This is a very common pitfall for non-pythonists https://www.toptal.com/python/python-class-attributes-an-overly-thorough-guide

Reply:
Thanks for the important resource!